recipes/extra-cmake-modules/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+mkdir build
+pushd build
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release     \
+      -Wno-dev \
+      ..
+
+make -j ${CPU_COUNT}
+ctest -E "ecm_setup_version-old.*|ECMPoQmToolsTest"
```

Comment:
Do you know why these fail? Can you run `ctest --verbose` to see if there's anything we can do to fix them? (Some packages rely on a git installation to automatically detect the version. This might be an issue here. Just a wild guess)

Reply:
I don't know, and I am not super interested in trying to fix old tests. This package is literally just a collection of cmake modules that gets copied into the approriate modules directory so that other things in KDE can build.  I have tested that other things farther down the stack (like karchive) do build and test locally on top of this package, so I don't think that these are important.  

If someone wants to fix them up, I think we can do that in the feedstock

Reply:
Also, there aree 72 other passing tests